linux_pipeline/Jenkinsfile_upstream_kernel

```diff
@@ -335,7 +340,6 @@ pipeline {
                         " -RGIdentifier '${LISAV2_RG_IDENTIFIER}'" +
                         " -TestPlatform 'Azure'" +
                         " -CustomKernel 'localfile:./scripts/package_building/${env.BUILD_NUMBER}-${env.BRANCH_NAME}-${env.KERNEL_ARTIFACTS_PATH}/*/${env.PACKAGE_TYPE}/*.${env.PACKAGE_TYPE}'" +
-                        " -OverrideVMSize '${LISAV2_AZURE_VM_SIZE_SMALL}'" +
```

Comment:
this line was on purpose here, to force the azure vm sizes. why do you remove it?

Reply:
Could you tell me the purpose that force the size?
Now LISAv2 supports economy mode, if the case not fail, the next case will be executed on the same VM. And in BVT test category, there are two cases, which are not supported in size A2, it will fail always.
  34 VERIFY-DEPLOYMENT-PROVISION-EPHEMERAL-MANAGED-DISK              ABORTED                 1.72  
  35 VERIFY-DEPLOYMENT-PROVISION-SRIOV                               ABORTED                 1.66